src/shared/utils.js

```diff
@@ -1,6 +1,6 @@
-function convertToSafeString(string){
-	return string.replace(/\W/g, '') // remove non-alphanumeric
-			.replace(/ /g, "_")      // replace spaces with underscores
+function convertToSafeString(string, seperator="_"){
```

Comment:
oh, btw, if you haven't seen this yet - this is a way to specify a default value for function arguments - in the same way as if we were doing object destructuring.

Reply:
old news mate 😏